src/test/resources/response/get_creditorinstitutions_ok2.json

```diff
@@ -1,7 +1,7 @@
 {
   "creditor_institutions": [
     {
-      "id_domain": "00168480242",
+      "organization_fiscal_code": "00168480242",
```

Comment:
according to this change we should update `CreditorInstitution` definition too. 
see `openapi/swagger.yaml`

Reply:
Right! swagger updated.